client/src/components/UI/ButtonGroup.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import GoogleLogin from './GoogleLogin';
+
+
+const styles = theme => ({
+  button: {
+    margin: theme.spacing.unit,
+    background: 'white',
+  },
+});
+
+class ButtonGroup extends React.Component {
+  state = { withOrWithoutLogin: false }
+
+  componentWillMount() {
```

Comment:
This can be removed.

Reply:
How?

Reply:
The method is empty right?

Reply:
ok, I thought you were talking about the state

Reply:
ok, I thought you were talking about the state